.gitignore

```diff
@@ -2,6 +2,8 @@
 world/
 players/
 resources/
+
+cmd/plugins/plugins.yaml
```

Comment:
we should keep some example plugins.yaml if this is going to be ignored. I think its better to revert this and handle it outside of the PR